ndbench-dynamodb-plugins/src/main/java/com/netflix/ndbench/plugin/dynamodb/DynamoDBKeyValue.java

```diff
@@ -79,19 +94,18 @@
     @Inject
     public DynamoDBKeyValue(AWSCredentialsProvider credential, DynamoDBConfigs config) {
         this.config = config;
-            String discoveryEnv = System.getenv(NdBenchConstants.DISCOVERY_ENV);
-            logger.error("Discovery Environment Variable: " + discoveryEnv);
+        String discoveryEnv = System.getenv(NdBenchConstants.DISCOVERY_ENV);
+        logger.error("Discovery Environment Variable: " + discoveryEnv);
         if (discoveryEnv == null || discoveryEnv.equals(NdBenchConstants.DISCOVERY_ENV_AWS)) {
             awsCredentialsProvider = credential;
         } else {
             awsCredentialsProvider = new ProfileCredentialsProvider();
             try {
-
-            awsCredentialsProvider.getCredentials();
```

Comment:
Should the credentials loading part be part of `init`?

Reply:
Can we discuss the location of credentials in an issue to resolve in a different PR? This is a non-essential spacing change.

Reply:
#144 opened

Reply:
👍